server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -611,6 +611,11 @@ void sc_osc_handler::open_udp_socket(ip::address address, unsigned int port) {
     else
         sc_notify_observers::udp_socket.open(udp::v4());
 
+    boost::asio::socket_base::receive_buffer_size recv_buffer_size(8 * 1024 * 1024);
```

Comment:
>Or is there a reason to not set the value for supernova?

I noticed that we already set send buffer size to 65536 in sclang and scsynth. I'm not sure why we chose 65536, probably because the field for packet size is 32 bits long in UDP header? However, doing this will likely reduce the send buffer size compared to the default on Linux. (On Linux, typical value is 212992. When you set to 65536, kernel will double it, so it is 131072.) Maybe the behavior is different on other OS?